central/notifiers/slack/slack.go

```diff
@@ -37,8 +37,8 @@ type slack struct {
 	*storage.Notifier
 	client *http.Client
 
-	namespaces namespaceDataStore.DataStore
-	mitreStore mitreDataStore.MitreAttackReadOnlyDataStore
+	namespaceProperties notifiers.NamespaceProperties
+	mitreStore          mitreDataStore.MitreAttackReadOnlyDataStore
```

Comment:
Will need changes after rebase (mitre store name)

Reply:
Can do, I'll wait for your changes and then rebase